SubmersedVR/VR/VRQuickSlots.cs

```diff
@@ -62,10 +62,13 @@ void Awake()
 
         new void Init(IQuickSlots newTarget)
         {
-            Mod.logger.LogInfo($"[nameof{this.GetType()}] Init on {newTarget}");
-            base.Init(newTarget);
-            ArangeIconsInCircle(wheelRadius);
-            OnSelect(this.target.GetActiveSlotID());
+            if (newTarget != null)
```

Comment:
Good catch. I think I would prefer 

```cs
if (newTarget == null)
  return;
```

in the beginning though. Looks a bit cleaner and avoids too many indentations.